examples/minimal_agent_production/streamlit_app.py

```diff
@@ -0,0 +1,285 @@
+        with st.container():
+            st.subheader("📊 Analysis Results")
+
+            # Metrics row
+            col1, col2, col3, col4 = st.columns(4)
+            with col1:
+                sentiment_emoji = {
+                    "positive": "😊",
+                    "negative": "😞",
+                    "neutral": "😐",
+                }.get(analysis.get("sentiment", "neutral"), "😐")
+                st.metric(
+                    "Sentiment",
+                    f"{sentiment_emoji} {analysis.get('sentiment', 'N/A').title()}",
+                )
+            with col2:
+                readability = analysis.get("readability_score", 0)
+                readability_label = (
+                    "Easy"
+                    if readability > 0.7
+                    else "Medium"
+                    if readability > 0.4
+                    else "Hard"
+                )
+                st.metric(
+                    "Readability", f"{readability:.2f} ({readability_label})"
+                )
+            with col3:
+                st.metric("Word Count", f"{analysis.get('word_count', 0):,}")
+            with col4:
+                st.metric(
+                    "Processing Time", f"{analysis.get('latency_ms', 0)} ms"
+                )
+
+            # Analysis type indicator
+            analysis_method = analysis.get("metadata", {}).get(
+                "analysis_method", "unknown"
+            )
+            if analysis_method == "llm":
+                st.info("🤖 Powered by AI (OpenAI GPT-4o-mini)")
+            else:
+                st.info("⚙️ Deterministic analysis (offline mode)")
+
+        # Summary section
+        with st.container():
+            st.subheader("📝 Summary")
+            st.markdown(f"*{analysis.get('summary', 'No summary available')}*")
+
+        # Keywords section
+        with st.container():
+            st.subheader("🔑 Keywords")
+            keywords = analysis.get("keywords", [])
+            if keywords:
+                # Display keywords in columns for better layout
+                keyword_cols = st.columns(min(len(keywords), 3))
+                for i, keyword in enumerate(keywords):
+                    with keyword_cols[i % 3]:
+                        st.markdown(f"🏷️ **{keyword}**")
+            else:
+                st.write("No keywords extracted")
+
+        # Document info
+        with st.expander("📄 Document Details"):
+            doc_info = analysis.get("document", {})
+            col1, col2 = st.columns(2)
+            with col1:
+                st.write(f"**Filename:** {doc_info.get('filename', 'N/A')}")
+                st.write(
+                    f"**Document Type:** {doc_info.get('document_type', 'N/A')}"
+                )
+            with col2:
+                st.write(
+                    f"**Analysis Type:** {doc_info.get('analysis_type', 'N/A')}"
+                )
+                st.write(f"**Model:** {analysis.get('model', 'N/A')}")
```

Comment:
The Document Details section tries to show `doc_info['analysis_type']`, which isn’t on `DocumentRequest`.
```suggestion
            with col2:
                st.write(f"**Analysis Method:** {analysis.get('metadata', {}).get('analysis_method', 'N/A')}")
                st.write(f"**Model:** {analysis.get('model', 'N/A')}")
```